utils/superset/db_name_change.py

```diff
@@ -0,0 +1,52 @@
+from io import IOBase
+import os
+import json
+import argparse
+import logging
+from pyexpat import ErrorString
+import sys
+
+
+DATABASE_NAME = "template_database_name"
+
+# create logger
+logging.basicConfig(level=logging.DEBUG)
+
+
+# file extension validation
+def validate_extension(file_name):
```

Comment:
I think this is unnecessary - if the file is json it doesn't really matter what the extension is, and just because the file has a .json at the end doesn't mean it really is.

Reply:
The thing is, the upload json form accepts the file with only the `.json` extension. There is a `accept=".json"` in type= file HTML tag. Hence we need to enforce the file extension.

Reply:
That's just on your output, though.

Reply:
Yes, can control it, but one can enter the other extensions and the file upload just ignores the other extensions, hence that validate function. Something like

```
>>python3 db_name_change.py pgss -i pgd_monitoring_template.json -o output.sh
INFO:root:Database name entered: pgss
ERROR:root:Unexpected error while writing to the file: File must have JSON extension
```